dev-docs/development-process.md

```diff
@@ -331,3 +334,36 @@ payload:
   maxRunTime: 60
 ...
 ```
+
+## Debugging Github integration locally
+
+Please refer to the [Github integration manual](https://docs.taskcluster.net/docs/manual/deploying/github) on how to set it up initially.
+
+Github integration relies on webhook events that will be sent to the endpoint defined in your Github application settings.
+To be able to test it locally, you'd need to setup a tunnel, so those API calls will reach your dev env.
+
+### Using ngrok to forward webhook events
+
+You can use <https://ngrok.com/> for this purpose:
+
+1. Sign up for a free account (doesn't work otherwise)
```

Comment:
@ahal I borrowed your steps and included it in the doc already. 
If you have something already, let me know, we can merge it

Reply:
Nice! There's also:

1. Creating an org if you don't have one you can associate the app with already (though maybe that should go in the github deployment docs)
2. Setting the requisite env vars in `docker-compose.override.yml` (the deploy docs don't mention docker-compose and I had to sort of guess what to do.. Luckily it was intuitive and worked out)
3. Creating a repo to install the app in incl. a `.tasckluster.yml` that can use the built-in / docker-compose workers
4. Adding scopes

I think I'll still write an end-to-end tutorial as a blog post, but will upstream things that make sense to the dev-docs.

Reply:
Right! Thanks, will add those

Reply:
btw, you can also create application without an org, at least the one I created for my dev purposes belongs to my account, not org.

Reply:
Oh, so you can! I somehow missed that while following https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app ><.. Thanks for the pointer!